tools/shared.py

```diff
@@ -1224,6 +1227,11 @@ def __getattr__(self, attr):
         raise AttributeError
 
     def __setattr__(self, attr, value):
+      for legacy_attr, fixed_values, error_message in self.attrs['REMOVED_SETTINGS']:
+        if attr == legacy_attr and value not in fixed_values:
+          logger.error('Invalid command line option -s ' + attr + '=' + str(value) + ': ' + error_message)
```

Comment:
this error is reporting that a legacy key is used with the wrong value, I think? if so, the error message could be more specific on that, and report the only currently supported value?

Reply:
I wanted to keep this side of the print very unstructured, so that the actual "meat" of the print will then be possible to freely formatted in src/settings.js side, depending on what is desired to be said about each item. If you combine this string with those in src/settings.js, I hope the prints should look readable?